### PR TITLE
fix: allow unmapping dimension

### PR DESCRIPTION
**Pull request by martinmaillard**
Removing the mapped dimension currently causes an error. This PR fixes that.

### CODE REVIEW

File: .changeset/fast-cars-talk.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"@cube-creator/core-api": patch
+---
+
+Fix removing link between dimension and managed dimension
```
Comment (tpluscode) [inline .changeset/fast-cars-talk.md:5]: I would remove this time. The whole feature has been unreleased so far, so there is technically no bug to fix

File: apis/core/lib/domain/dimension-mapping/update.ts [hunk 2]
```diff
@@ -29,7 +29,7 @@ export async function update({
     throw new error.BadRequest('Unexpected value of schema:about')
   }
 
-  if (!managedDimension.equals(dimensionMappings.managedDimension)) {
+  if (!managedDimension || !managedDimension.equals(dimensionMappings.managedDimension)) {
```
Comment (tpluscode) [inline apis/core/lib/domain/dimension-mapping/update.ts:32]: Worthy of adding a unit test?

File: .changeset/fast-cars-talk.md (hunk 1 above)
Comment (martinmaillard) [reply .changeset/fast-cars-talk.md:5]: Yeah... that's why I'm not a fan of semi-automated changelogs ;)
Will remove

File: apis/core/lib/domain/dimension-mapping/update.ts (hunk 2 above)
Comment (martinmaillard) [reply apis/core/lib/domain/dimension-mapping/update.ts:32]: Probably yes. I tried and then got lazy. Is there a reason why the "when" of these unit tests is in the `beforeEach`? That seems really wrong and makes it hard to add new test cases in the same `describe`.

Comment (tpluscode) [reply apis/core/lib/domain/dimension-mapping/update.ts:32]: Yea, that doesn't make sense. Especially that there is only one test case in that `describe`.

Definitely move the `await update()` inside the `it()`

**Review by tpluscode — APPROVED** [review]

**Pull request merged**